Packs/PaloAltoNetworks_IoT3rdParty/Scripts/GetCiscoISEActiveInstance/GetCiscoISEActiveInstance.py

```diff
@@ -0,0 +1,49 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+# Run cisco-ise-get-nodes command on all ISE instances (primary and secondary)
+# Primary and Secondary instances should be passed as an argument to the script (Using)
```

Comment:
please consider refactoring with a `main()` entry point, it is quite hard to track this code.

Reply:
added main()